.gitignore

```diff
@@ -4,3 +4,4 @@ composer.phar
 phpcs.xml
 phpstan.neon
 vendor/
+.idea/
```

Comment:
Hi, for this is not project related you should add it to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
Didn't know that existed, thanks!